node/node.go

```diff
@@ -55,10 +55,6 @@ type Node struct {
 	// p2p protocols
 	PubSub *pubsub.PubSub
 	// BlockService provides access to the node's Block Service
-	// TODO @renaynay: I don't like the concept of storing individual services on the node,
-	// TODO maybe create a struct in full.go that contains `FullServices` (all expected services to be running on a
-	// TODO full node) and in light, same thing `LightServices` (all expected services to be running in a light node.
-	// TODO `FullServices` can include `LightServices` + other services.
```

Comment:
why did you remove this TODO? services are still stored not uniformly on the node: 

```
	BlockServ  *block.Service  `optional:"true"`
	ShareServ  share.Service   // not optional
	HeaderServ *header.Service // not optional
```

Reply:
> services are still stored not uniformly on the node.

TODO text is not related to the quote below, but to the fact that they are _stored on__ the node.

> why did you remove this TODO?

For you, you see this removal and create an appropriate issue. The TODO is not really a TODO, but an opinion spread onto four lines. Create an issue with the concern if still appropriate.